Mathlib/FieldTheory/Normal.lean

```diff
@@ -359,6 +359,43 @@ theorem AlgEquiv.restrictNormalHom_surjective [Normal F K₁] [Normal F E] :
     Function.Surjective (AlgEquiv.restrictNormalHom K₁ : (E ≃ₐ[F] E) → K₁ ≃ₐ[F] K₁) := fun χ =>
   ⟨χ.liftNormal E, χ.restrict_liftNormal E⟩
 
+@[simp]
+theorem AlgEquiv.restrictNormalHom_id (F K : Type*)
+    [Field F] [Field K] [Algebra F K] [Normal F K] :
+    AlgEquiv.restrictNormalHom (F := F) (K₁ := K) K = MonoidHom.id (K ≃ₐ[F] K) := by
+  ext f x
+  dsimp [restrictNormalHom]
+  apply (algebraMap K K).injective
```

Comment:
When calling a non-terminal `dsimp`, try to use `dsimp?` to get the `dsimp only` version.